Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabUiHandler.h

```diff
@@ -89,8 +91,9 @@ namespace AzToolsFramework
         QString m_prefabEditCloseIconPath = QString(":/Entity/prefab_edit_close.svg");
 
         inline static const QColor s_overrideIconBackgroundColor = QColor("#444444");
-        inline static const QPoint s_overrideIconOffset = QPoint(10, 10);
-        inline static const int s_overrideIconSize = 5;
-        QIcon s_overrideIcon = QIcon(QString(":/Entity/entity_overridden.svg"));
+        inline static const QPoint s_overrideIconOffset = QPoint(12, 12);
+        inline static const int s_overrideIconSize = 8;
+        QIcon m_overrideIconReplace = QIcon(QString(":/Entity/entity_overridden.svg"));
```

Comment:
nit: svg can be renamed to entity_modified_as_override to keep the naming consistent.
Similarly, the icon names can be m_editEntityOverrideIcon and m_addEntityOverrideIcon